components/public-api/typescript-common/src/public-api-converter.ts

```diff
@@ -1059,7 +1059,45 @@ export class PublicAPIConverter {
         }
     }
 
-    fromTimeoutSettings(timeoutSettings: TimeoutSettings): TimeoutSettingsProtocol {
+    fromOrganizationSettings(settings: PlainMessage<OrganizationSettings>): OrganizationSettingsProtocol {
```

Comment:
This is the field translation, untangled from validation etc. from the `organization-service-api.ts` handler.